packages/sitecore-jss-nextjs/package-lock.json

```diff
@@ -6274,63 +4276,62 @@
         "semver": "^5.4.1"
       }
     },
-    "neo-async": {
-      "version": "2.6.2",
-      "resolved": "https://registry.npmjs.org/neo-async/-/neo-async-2.6.2.tgz",
-      "integrity": "sha512-Yd3UES5mWCSqR+qNT93S3UoYUkqAZ9lLg8a7g9rimsWmYGK8cVToA4/sF3RrshdyV3sAGMXVUmpMYOw+dLpOuw==",
-      "dev": true
-    },
     "next": {
-      "version": "10.0.3",
-      "resolved": "https://registry.npmjs.org/next/-/next-10.0.3.tgz",
-      "integrity": "sha512-QYCfjZgowjaLUFvyV8959SmkUZU/edFgHeiXNtWDv7kffo/oTm891p0KZAkk5cMIHcsDX3g3UuQdw/zmui783g==",
+      "version": "10.2.0",
```

Comment:
Looks like next was updated to 10.2? Isn't that risky without updating the package.json version as well to match?
I've avoided updating next specifically with the last couple security audits I did (all low severity from what I recall) since we've got an upgrade to latest next coming up with v19.

Reply:
Hm...yes, I removed 10.2.0 from package.json - previously it automatically updated it, I will revert it

Reply:
Done